build-system/release-tagger/create-release.js

```diff
@@ -170,6 +170,7 @@ function _createBody(head, base, prs) {
  
```

Comment:
The rule can be disabled in `build-system/.eslintrc.js`

Reply:
Why would we do that?

Reply:
I think they're all difficult to read, but the only reason I care is the difficulty to create transforms.